apps/fishing-map/features/map/map.slice.ts

```diff
@@ -24,10 +24,11 @@ import { selectUserLogged } from 'features/user/user.slice'
 import { getRelatedDatasetByType, getRelatedDatasetsByType } from 'features/datasets/datasets.utils'
 
 export const MAX_TOOLTIP_LIST = 5
-export const MAX_VESSELS_LOAD = 200
+export const MAX_VESSELS_LOAD = 150
```

Comment:
I had some error 431 so I decreased this number a bit

Reply:
👍